mem/buffers.go

```diff
@@ -194,19 +197,19 @@ func (b *buffer) read(buf []byte) (int, Buffer) {
 	return n, b
 }
 
-// String returns a string representation of the buffer. May be used for
-// debugging purposes.
 func (b *buffer) String() string {
```

Comment:
mmm, interesting that this one is removed.

Reply:
yeah I was removing the docstrings for non exported structs